workflow/scripts/calc-precision-recall.py

```
@@ -16,41 +16,50 @@
-            counter[bin] += 1
-        else:
-            counter += 1
-        return counter
+            r = list(other_record.fetch(current_record.contig, current_record.start, current_record.stop))
+            if len(r) > 0:
+                r = r[0]
+                if fp:
+                    vaf = r.info[self.vaf_field_name_query] if self.vaf_field_query == "INFO" else r.samples[0][self.vaf_field_name_query]
+                else:
+                    vaf = r.info[self.vaf_field_name_truth] if self.vaf_field_truth == "INFO" else r.samples[0][self.vaf_field_name_truth]
+                if type(vaf) == tuple:
```

Ruff (0.12.2)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)